### PR TITLE
explain and profile

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/17.query-tuning-statements/1.explain-and-profile.md [hunk 1]
```diff
@@ -1,97 +1,99 @@
...
+|`name`|The name of the `operator`.|
+|`dependencies`|The ID of the `operator` that the current `operator` depends on.|
+|`profiling data`|The content of the execution profile. `ver` is the version of the `operator`. `rows` shows the number of rows to be output by the `operator`. `execTime` shows the execution time of `action`. `totalTime` is the sum of the execution time, the system scheduling time, and the queueing time.|
+|`operator info`|The detailed information of the `operator`.|
 
-## Format "dot"
+## The `dot` format
 
-You can use the `format="dot"` option to output the return message in the DOT language, and then use Graphviz to generate a graph of the plan.
+You can use the ``format="dot"`` option to output the return message in the `dot` language, and then use Graphviz to generate a graph of the plan.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/17.query-tuning-statements/1.explain-and-profile.md:92]: doubled `

**Review by whitewum — APPROVED** [review]

File: docs-2.0/3.ngql-guide/17.query-tuning-statements/1.explain-and-profile.md [hunk 2]
```diff
@@ -1,97 +1,99 @@
...
 |-|-|
-|`id`|Indicates the ID of the operator.|
-|`name`|Indicates the name of the operator.|
-|`dependencies`|Shows the ID of the operator that the current operator depends on.|
-|`profiling data`|Shows the execution profile. `ver` is the version of the operator, which you can use to identify loops; `rows` shows the number of rows to be output by the operator; `execTime` shows the execution time only; `totalTime` contains the execution time and the system scheduling and queueing time.|
-|`operator info`|Shows the detailed information of the operator.|
+|`id`|The ID of the `operator`.|
+|`name`|The name of the `operator`.|
+|`dependencies`|The ID of the `operator` that the current `operator` depends on.|
+|`profiling data`|The content of the execution profile. `ver` is the version of the `operator`. `rows` shows the number of rows to be output by the `operator`. `execTime` shows the execution time of `action`. `totalTime` is the sum of the execution time, the system scheduling time, and the queueing time.|
```
Comment (foesa-yang) [inline docs-2.0/3.ngql-guide/17.query-tuning-statements/1.explain-and-profile.md:87]: `action`;

**Review by whitewum — DISMISSED** [review]

File: docs-2.0/3.ngql-guide/17.query-tuning-statements/1.explain-and-profile.md (hunk 2 above)
Comment (izhuxiaoqing) [reply docs-2.0/3.ngql-guide/17.query-tuning-statements/1.explain-and-profile.md:87]: Semicolons are seldom used in our docs, so I guess the full stop here is ok.

File: docs-2.0/3.ngql-guide/17.query-tuning-statements/1.explain-and-profile.md [hunk 3]
```diff
@@ -109,6 +111,6 @@ Execution Plan
 ---------------------------------------------------------------------------------------------------------------------------------------------  -------------
 ```
 
-Transformed into a Graphviz graph, it is as follows:
+If you want to convert the `dot` language of the above example into a Graphviz graph, see the following figure.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/17.query-tuning-statements/1.explain-and-profile.md:114]: Inaccurate translation. Check the source.

**Review by randomJoe211 — APPROVED** [review]

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**